src/main/java/duke/exceptions/DukeException.java

```diff
@@ -0,0 +1,26 @@
+package duke.exceptions;
+
+
+/**
+ * Duke Exception class.
```

Comment:
Can give slightly more elaboration here.

Personally, i put "Throws custom exception commands if something goes wrong when processing user input." in my own DukeException so hope that it might help you as well on elaborating